lib/CMakeLists.txt

```diff
@@ -108,6 +110,8 @@ if(ENABLE_STATIC_LIB)
     )
   target_compile_definitions(nghttp3_static PUBLIC "-DNGHTTP3_STATICLIB")
 
+  target_include_directories(nghttp3_static INTERFACE $<INSTALL_INTERFACE:include>)
+
```

Comment:
nit: to stay in sync with the same formula in nghttp2 and also the shared section in this file, I suggest moving these two lines in between `set_targe_properties` and `target_compile_definitions`. Meaning two lines higher than now.

Reply:
done.